tests/functional/team_agent/team_agent_functional_test.py

```diff
@@ -236,3 +236,35 @@ def test_add_remove_agents_from_team_agent(run_input_map, delete_agents_and_team
 
     team_agent.delete()
     new_agent.delete()
+
+
+def test_team_agent_tasks(delete_agents_and_team_agents):
```

Comment:
Can we have single agent with tasks and run it? If yes, we should create a functional test for that. If not, we should restrict that when creating the single agent.

Reply:
tasks in single agents are irrelevant. They are just used on teams.